docs/modules/sql/pages/mapping-to-mongo.adoc

```diff
@@ -165,6 +163,73 @@ DATA CONNECTION myMongo
 ----
 Notice that there is no mention of `TYPE MONGO` this time; it’s automatically assumed by the SQL engine when you provide MongoDB data connection. This works with both schema provided or not.
 
+
+[NOTE]
+====
+You have to specify the database name using one of the following options:
+
+* add `OPTIONS ('database' = 'myDatabase')` in `CREATE DATA CONNECTION`
+* add `OPTIONS ('database' = 'myDatabase')` in `CREATE MAPPING`
+* use the full external name, e.g., `CREATE MAPPING people EXTERNAL NAME myDatabase.people (...)`
+====
+
+== Supported Object Types
+
+For given connector there may be 1 or more object types. For Mongo SQL Connector we have 2 object types:
+
+ - `Collection` - represents batch raeding from given collection. This is the default object type.
+ - `ChangeStream` - represents reading a stream of events.
+
+To change object type you want to use, you have to append `OBJECT TYPE X` after `DATA CONNECTION` / `TYPE`, for example:
+
+[tabs]
+====
++
+Using Data Connection::
+--
+[source,sql]
+----
+CREATE MAPPING people (
+    firstName VARCHAR(100),
+    lastName VARCHAR(100),
+    age INT
+)
+DATA CONNECTION myMongo
+OBJECT TYPE ChangeStream
+OPTIONS (...)
+----
+--
++
+Using TYPE::
+--
+[source,sql]
+----
+CREATE MAPPING people (
+firstName VARCHAR(100),
+lastName VARCHAR(100),
+age INT
+)
+TYPE Mongo
+OBJECT TYPE ChangeStream
+OPTIONS (...)
+----
+--
+====
+
+== Field mappings
+
+Object type `Collection` reads columns to be the same as properties in Mongo collection.
+
+For `ChangeStream` it's more complicated: the ChangeStream objects have Mongo collection's properties prefixed with `fullDocument.`,
+There are also some predefined, top-level (without `fullDocument.` prefix) columns
+
+ - `operationType STRING` - operation that triggered this Change event, e.g. `insert`
+- `resumeToken STRING` - resume token associated with given change stream event.
+- `wallTime DATE_TIME` - wall time of the event.
```

Comment:
Not sure this is very useful if you don't know what "wall time" is. Could you give a fuller description?